packages/trader/src/AppV2/Components/TradeParameters/Duration/duration.tsx

```diff
@@ -168,7 +164,7 @@ const Duration = observer(({ is_minimized }: TDurationProps) => {
                 label={<Localize i18n_default_text='Duration' key={`duration${is_minimized ? '-minimized' : ''}`} />}
                 value={getInputValues()}
                 noStatusIcon
-                disabled={isMarketClosed(activeSymbols, symbol)}
```

Comment:
@ahmadtaimoor-deriv please, help to review this change as you were working with duration component and I might breake something

Reply:
`activeSymbols` from hook is basically something that will be set to the store, while `is_market_closed` from trade store is based on `active_symbols` from store directly (as I understand).
Please, @henry-deriv, correct me in case